RestSharpClient/Utils/JsonSerializer.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+namespace RestSharpClient.Utils
```

Comment:
please use file scoped namespaces

Reply:
Changed to file scoped namespaces at all the files.